service/indexer/indexer.go

```diff
@@ -384,19 +403,40 @@ func (idx *indexer) addBlockMaybeReorg(ctx context.Context, header *types.Header
 		}
 		blocks = append(blocks, block)
 	}
-	logger.Trace("Reorg tracing: Stop", "at", block.Number(), "hash", block.Hash().Hex())
 	branchBlock := block
 	blocksToInsert = append(blocksToInsert, blocks...)
+	logger = logger.New("branchNumber", branchBlock.Number(), "branchHash", branchBlock.Hash().Hex())
 
 	// Now atomically update the reorg'ed blocks
```

Comment:
update comment:
`Now delete the reorg'ed blocks and rely on top-level sync to insert new blocks`

Reply:
OK